productsCtrl.js

```diff
@@ -1,12 +1,23 @@
-var app = require('./index.js');
+// var app = require('./index.js');
+let db;
 
 module.exports = {
-
-  create: function(app, name, desc, price, url){
-    db.create_product([name, desc, price, url], function(err, product){
-        console.log('this is product', product)
+  setDb: function(database) {
+    db = database;
+  },
+  create: function(req, res) {
+    const { name, desc, price, url} = req.body;
```

Comment:
this is es6 destructuring. in the req.body (like in postman) i passed an object: 
```
{
  name: "whatever",
  desc: "yep",
  price: 2042,
  url: "ballinhard.com" 
}

this enables us to pass each as separate variables when we call `db.create_product`